interface/billing/sl_receipts_report.php

```diff
@@ -413,38 +433,49 @@ function sel_diagnosis() {
       // If a doctor was specified.
       if ($form_doctor) $query .= " AND u.id = '$form_doctor'";
       ***************************************************************/
+      $sqlBindArray = array();
       $query = "SELECT a.pid, a.encounter, a.post_time, a.code, a.modifier, a.pay_amount, " .
         "fe.date, fe.id AS trans_id, fe.provider_id AS docid, fe.invoice_refno, s.deposit_date, s.payer_id, " .
-        "b.provider_id " .
+        "b.provider_id, concat(lname, ' ', fname) as 'pat_fulname' " .
         "FROM ar_activity AS a " .
         "JOIN form_encounter AS fe ON fe.pid = a.pid AND fe.encounter = a.encounter " .
         "LEFT OUTER JOIN ar_session AS s ON s.session_id = a.session_id " .
+        "LEFT OUTER JOIN patient_data ON patient_data.pid = a.pid " .
```

Comment:
Since this is a large query would follow the standard on it and assign patient_data a letter:
``````
"LEFT OUTER JOIN patient_data as p ON patient_data.pid = a.pid " .